### ISSUE TITLE
[Question]: Can I provide a Chromium channel when using PageTest in unit?

### ISSUE THREAD

**Issue by jevgenijusmarinuskinas** [microsoft/playwright-dotnet#1730]
### Your question

Hello,

I can specify BROWSER and HEADED through the means of environment variables, to be used with a PageTest base class in my code. I've looked through the playwright sources, and was unable to find an environment variable for a Chromium channel. Is it possible to somehow control it in this scenario (for example, by setting some property, etc.)?

**Comment by avodovnik** [comment]
No, unfortunately this isn't possible at the moment. I'll convert this into a feature request though, and we can explore adding it.

**Comment by PavanMudigondaTR** [comment]
I have the same question and request. Playwright supports browser_channel environment variable for Python, Java, JS/TS. I am not seeing it working in dotnet. I am struggling to figure out how to run Github Actions for Chrome and MSEdge.
